cmd/influxd/run.go

```diff
@@ -23,7 +23,7 @@ import (
 	"github.com/influxdb/influxdb/udp"
 )
 
-func Run(config *Config, join, version string) (*messaging.Broker, *influxdb.Server) {
+func Run(config *Config, join, version string) (*messaging.Broker, *influxdb.Server, *raft.Log) {
```

Comment:
Agreed.  We started down that path in https://github.com/influxdb/influxdb/pull/2118.  We're moving in that direction but planning to do that in a separate PR.